--- openslp-1.0.2/test/Makefile.am~	Thu Jun 21 17:29:15 2001
+++ openslp-1.0.2/test/Makefile.am	Sat Sep 15 13:46:47 2001
@@ -56,8 +56,8 @@
 clean-local:
 	-rm -f *.output
 
+# We have to manually copy files in the VPATH directories. 
 dist-hook:
-	# We have to manually copy files in the VPATH directories. 
 	@for d in `echo $(VPATH) | sed 's/:/ /g'`; do             \
             if test $$d != $(srcdir); then                        \
                 cp -pr $$d $(distdir)/`basename $$d`;             \
