--- openslp-1.2.1/configure.in.orig	2005-02-15 21:06:02.000000000 +0100
+++ openslp-1.2.1/configure.in	2005-08-15 11:42:58.633764528 +0200
@@ -129,9 +129,6 @@
     if test X"$debug" = X"yes"; then
        CFLAGS="$CFLAGS -Werror"
     fi
-    if test X"$debug" = X"no"; then 
-       OPTFLAGS="-O3"
-    fi
     
 elif $CC -V 2>&1 | grep "WorkShop Compilers"; then
     dnl Allow C++ style comments
@@ -170,9 +167,9 @@
 dnl ***********************************************************************
 dnl Checks for libraries.
 dnl ***********************************************************************
-AC_CHECK_LIB(resolv, inet_aton)
-AC_CHECK_LIB(socket, main)
-AC_CHECK_LIB(nsl, gethostbyname)
+AC_SEARCH_LIBS(inet_aton, resolv)
+AC_SEARCH_LIBS(socket, socket)
+AC_SEARCH_LIBS(gethostbyname, nsl)
 dnl AC_CHECK_LIB(m, main)
 dnl AC_CHECK_FUNCS(ceil log10 strncasecmp strcasecmp )
 AC_CHECK_FUNCS(strncasecmp strcasecmp)
