--- openslp-1.1.4/configure.in.orig	2003-07-15 09:44:55.000000000 +0200
+++ openslp-1.1.4/configure.in	2003-07-20 13:49:36.000000000 +0200
@@ -91,6 +91,15 @@
 fi
 
 dnl ***********************************************************************
+dnl Checks for programs.
+dnl ***********************************************************************
+AC_PROG_CC
+AC_PROG_INSTALL
+AM_PROG_LIBTOOL
+AM_PROG_LEX
+AC_PROG_YACC
+
+dnl ***********************************************************************
 dnl Enable security support 
 dnl ***********************************************************************
 AC_ARG_ENABLE(slpv2-security,
@@ -108,15 +117,6 @@
 
 
 dnl ***********************************************************************
-dnl Checks for programs.
-dnl ***********************************************************************
-AC_PROG_CC
-AC_PROG_INSTALL
-AM_PROG_LIBTOOL
-AM_PROG_LEX
-AC_PROG_YACC
-
-dnl ***********************************************************************
 dnl Set C/C++ compiler specific warning/optimization flags
 dnl ***********************************************************************
 if test X"$GCC" = X"yes"; then        
