--- openslp-1.2.1/configure.ac.orig	2005-02-15 21:06:02.000000000 +0100
+++ openslp-1.2.1/configure.ac	2005-08-15 11:42:58.633764528 +0200
@@ -29,9 +29,9 @@
 #
 # Checks for libraries
 #
-AC_CHECK_LIB([resolv], [inet_aton])
-AC_CHECK_LIB([socket], [socket])
-AC_CHECK_LIB([nsl], [gethostbyname])
+AC_SEARCH_LIBS([inet_aton], [resolv])
+AC_SEARCH_LIBS([socket], [socket])
+AC_SEARCH_LIBS([gethostbyname], [nsl])
 AC_CHECK_LIB([m], [log10])
 AC_CHECK_LIB([pthread], [pthread_create])
 
@@ -213,11 +213,6 @@
   #   disable optimization for debug builds
   #   enable high-level optimizations for non-debug builds
   CFLAGS="$CFLAGS -Wall $GCC_OS_FLAGS"
-  if test "x${debug}" = xyes; then 
-    OPTFLAGS="-O0"
-  else
-    OPTFLAGS="-O2"
-  fi
 elif $CC -V 2>&1 | grep "WorkShop Compilers"; then
   # Solaris Workshop: enable optimizations for debug builds
   if test "x${debug}" = xno; then
